spec.bs

```diff
@@ -639,6 +641,21 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=auction config/per buyer experiment group ids=].
   1. If |perBuyerExperimentGroupIds| is not null and |perBuyerExperimentGroupIds|[|buyer|] exists:
     1. Set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
+  1. Let |buyerGroupLimit| be |allBuyersGroupLimit|.
+  1. Let |perBuyerGroupLimits| be |auctionConfig|'s
+    [=auction config/per buyer group limits=].
+  1. If |perBuyerGroupLimits| is not null and |perBuyerGroupLimits|[|buyer|] exists:
+    1. Set |buyerGroupLimit| to |perBuyerGroupLimits|[|buyer|].
+  1. Let |igs| be a new [=list=] of [=interest groups=].
+  1. [=map/For each=] signalsUrl -> |perSignalsUrlGenerator| of |perBuyerGenerator|:
+    1. [=map/For each=] joiningOrigin -> |groups| of |perSignalsUrlGenerator|:
+      1. [=list/Extend=] |igs| with |groups|.
+  1. [=list/Sort=] |igs| in descending order, with |a| being less than |b| if |a|'s

```

Comment:
Can you link to https://infra.spec.whatwg.org/#list-sort-in-descending-order?

Reply:
I wanted to do that...but I had no idea how.  Do I add an anchor dfn link at the top to that and then make the "in descending order" just link to that? how do I find what pieces of "list" I can link to automatically? e.g. list/Sort

Reply:
On [webdex](https://dontcallmedom.github.io/webdex/l.html#list%40%40infra%25%25dfn) you'll see the "Related terms" section which shows the exported terms associated with "list".

I would do either:
 1. What this reference does (https://wicg.github.io/attribution-reporting-api/#ref-for-list-sort-in-descending-order) which links "sorting" to that term, and doesn't link anything to "in descending order"
 2. Or just write `[=list/Sort in descending order=] |igs|, with |a| [...]`

Reply:
Ah cool.  That webdex thing is neat.